buildout.d/versions.cfg

```diff
@@ -2,6 +2,7 @@
 setuptools = 8.3
 zc.buildout = 2.2.5
 zc.recipe.egg = 2.0.1
+plone.app.widgets = 1.8.0
```

Comment:
@tisto  Because if we are not pining it then p.a.widget 2.0.0 is called and that creates problem as that latest version of p.a.widget was calling CMFPlone5.0.b.dev and that makes travis sad.